sonarts-core/docs/rules/no-big-function.md

```diff
@@ -0,0 +1,11 @@
+# no-big-function
+
+A function that grows too large tends to aggregate too many responsibilities.
+
+Such functions inevitably become harder to understand and therefore harder to maintain. 
+
+Above a specific threshold, it is strongly advised to refactor into smaller functions which focus on well-defined tasks.
+
+Those smaller functions will not only be easier to understand, but also probably easier to test.
+
```

Comment:
@vilchik-elena should configuration be documented here?

Reply:
@saberduck Indeed!